recipes/sonicparanoid/meta.yaml

```diff
@@ -32,17 +32,17 @@ requirements:
     - setuptools >=57.4.0
     - wheel >=0.37.0
     - Cython >=0.29.24
+  run:
+    - pip >=21.2.4
```

Comment:
Is `pip` really a runtime requirement?

Reply:
Hello @marcelm, technically it should not be needed, nevertheless when I tested on the artifacts on a Ubuntu VM the execution was failing because some of the modules were missing.  
I have tested the latest [artifacts](https://170403-42372094-gh.circle-artifacts.com/0/tmp/artifacts/packages) and everything is working fine.  
Thanks a lot for improving the recipe.